components/script/script_runtime.rs

```diff
@@ -826,7 +832,46 @@ impl StreamConsumer {
 }
 
 #[allow(unsafe_code)]
-unsafe extern "C" fn report_stream_error_callback(_cx: *mut JSContext, error_code: usize) {
+unsafe extern "C" fn ConsumeStreamCallback(
```

Comment:
Let's call this `consume_stream`, and add a comment like `/// <https://webassembly.github.io/spec/web-api/#compile-a-potential-webassembly-response` to be clear that we're implementing a particular specification.

Reply:
Done